[CDS-89172]: Added DISABLE_CF_APP_LOG_STREAMING env var to not show t… (#5088)
harness/developer-hub: release-notes/delegate.md

## January 2024

### Version 24.01.82109

#### Hotfix

> - In this release, we address the issue of application logs being printed in TAS deployment execution logs. To enhance control over this behavior, we introduce a new environment variable: `DISABLE_CF_APP_LOG_STREAMING`. Setting this variable to `true` will redact all application logs, providing users with more flexibility in managing log visibility. (CDS-89172)

Reviewer: ```suggestion
- Application logs were printed in TAS deployment execution logs. (CDS-89172)

   Harness added a new environment variable `DISABLE_CF_APP_LOG_STREAMING` to enhance control over this behavior. Setting this variable to `true` will redact all application logs, providing users with more flexibility in managing log visibility. 
```
